cluster-autoscaler/cloudprovider/gce/templates.go

```diff
@@ -515,10 +553,12 @@ func extractOperatingSystemDistributionFromKubeEnv(kubeEnv string) OperatingSyst
 		return OperatingSystemDistributionWindowsSAC
 	case string(OperatingSystemDistributionCOS):
 		return OperatingSystemDistributionCOS
-	case string(OperatingSystemDistributionCOSContainerd):
-		return OperatingSystemDistributionCOSContainerd
-	case string(OperatingSystemDistributionUbuntuContainerd):
-		return OperatingSystemDistributionUbuntuContainerd
+	// Depricated
+	case "cos_containerd":
```

Comment:
Maybe log a warning when _containerd distribution is used? We could eventually just remove it.